### ISSUE TITLE
Feature request: slurp into named variable

### ISSUE THREAD

**Issue by jbrains** [01mf02/jaq#52]
This is not urgent, but it would be very nice.

I would prefer to slurp the contents of files directly into variables, rather than using the position that a file happens to appear in the stream. I find it nicer to work with keyword parameters than to work with positional parameters. :)

I admit that this is perhaps largely a matter of personal preference. I'm curious how the crowd feels about this. Certainly, I can do everything I need with the current behavior of slurp, but this feature would make the resulting `jaq` code easier to follow.

**Comment by wader** [comment]
Maybe copy what jq has?
```
  --slurpfile a f  set variable $a to an array of JSON texts read from <f>;
  --rawfile a f    set variable $a to a string consisting of the contents of <f>;
```

**Comment by 01mf02** [comment]
I definitely see the use of this. I implemented your suggestions in 2ce0e62d4dbbbc9e8939e73fbd2a14e5045594e9.
